website/docs/components/dropdown/partials/guidelines/guidelines.md

```diff
@@ -294,6 +295,7 @@ Use icons consistently and when they reinforce the content.
   <Doc::ListContainer class="hds-dropdown__list">
     <Hds::Dropdown::ListItem::Interactive @text="Rename cluster" @color="action" @icon="edit" />
     <Hds::Dropdown::ListItem::Interactive @text="Restore cluster" @color="action" @icon="reload" />
+    <Hds::Dropdown::ListItem::Interactive @text="Reference cluster" @color="action" @icon="github" @trailingIcon="external-link" />
```

Comment:
@didoo for some reason this isn't showing up in the example. Clearly I'm missing something.

Reply:
it's there, [have a peek](https://hds-website-git-dropdown-listitem-interactive-72c92b-hashicorp.vercel.app/components/dropdown#icon-usage). if you tested this on your machine, remember that since v2 changes you'd need to run `yarn start` in `packages/components` alongside with running the `website` with `ember s` – this is to make sure the latest changes in the components library are compiled and ready to be used on the website.

Reply:
I'm a goof ball lol It's ALWAYS this step that gets me lol Thanks for the reminder.